test/src/main/java/org/openjdk/skara/test/TestPullRequest.java

```diff
@@ -350,4 +350,9 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(super.hashCode(), headHash, sourceRef, targetRef, draft);
     }
+
+    @Override
+    public boolean diffLimited() {
+        return false;
+    }
```

Comment:
You could write an automatic test for PullRequestBot if you give the test the ability to control if the diff returned from TestPullRequest is limited or not.